kitchen/src/kitchen/core.clj

```diff
@@ -262,6 +262,13 @@
   {:status intended-status
    :body "Health check returned bad status"})
 
+(defn sleep-handler
+  "Sleeps for given number of milliseconds"
+  [sleep-ms]
+  (Thread/sleep sleep-ms)
+  {:status 400
```

Comment:
Do we always want this to return 400? Aren't there scenarios where we'd want to sleep but get back, say, 200?